components/brave_vpn/features.cc

```diff
@@ -25,6 +25,9 @@ BASE_FEATURE(kBraveVPNLinkSubscriptionAndroidUI,
              "BraveVPNLinkSubscriptionAndroidUI",
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+BASE_FEATURE(kBraveVPNDnsProtection,
+             "BraveVPNDnsProtection",
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
this should only be enabled by default on windows

Reply:
actually the mix of buildflag and featurelist is inconsistent right now so I think we should put this entire feature flag behind IS_WIN. That will ensure that any references to it are necessarily guarded by both ENABLE_BRAVE_VPN and IS_WIN or they won't compile. It's also not relevant to any other platform anyway